packages/ui/src/theme/tokens/components/fileUploader.ts

```diff
@@ -12,7 +12,8 @@ type BaseDropZoneTokens<OutputType> = DesignTokenProperties<
 
 export interface FileUploaderTokens<OutputType extends OutputVariantKey> {
   dropzone?: DesignTokenProperties<
-    'gap' | 'paddingBlock' | 'paddingInline' | 'textAlign'
+    'gap' | 'paddingBlock' | 'paddingInline' | 'textAlign',
+    OutputType
```

Comment:
Note: this is a bug because with the 2nd arg all these properties become required

Reply:
Did you already fix this in StorageManager, or did you mean to fix this in FileUploader?

Reply:
StorageManger basically just extends this type so it is fixed in both :)

Reply:
So do we need to rename this type? When we delete the FileUploader tokens in next release, won't StorageManager break?

Reply:
When we do that breaking change we will inline that type into the storagemanager type